_docs_v7/Gradients-Limiters.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Gradients and Limiters
+permalink: /docs_v7/Gradients-Limiters/
+---
+
+This page lists the gradient of the space computation methods and the limiter functions in SU2 as well as their associated options, it is not meant as a detailed theory guide but some application guidance is given nonetheless. The options listed here do not apply to the high order DG solver.
+
+---
+
+
+---
+
+## Gradient Computation ##
+The numerical method for the computation of the spatial gradients used for viscous fluxes and source terms is specified by the `NUM_METHOD_GRAD` field. The list of available options is given below:
+- `GREEN_GAUSS`: Classic gradient reconstruction based on the Green-Gauss theorem (edge-based).
+- `WEIGHTED_LEAST_SQUARES`: Compute the gradient of a field using inverse-distance-weighted approximation.
+The default option is set to `WEIGHTED_LEAST_SQUARES`.
+
+The spatial gradients method used only for upwind reconstruction is specified by the `NUM_METHOD_GRAD_RECON` field. An additional method is available:
+- `LEAST_SQUARES`: Compute the gradient of a field using unweighted Least- Squares approximation.
+
+If the `NUM_METHOD_GRAD_RECON` field is left empty or set to `NONE` it defaults to `NUM_METHOD_GRAD`.
+
+Thin Shear Layer gradient reconstruction is always used for the construction of the Jacobian.
+
+## Limiters ##
+SU2 implements limiter functions to prevent the generation of oscillations when using upwind spatial discretisations. These are specified by the config field `SLOPE_LIMITER_FLOW`. The available options are:
+- `NONE`                 : No limiter
+- `VENKATAKRISHNAN`      : Slope limiter using Venkatakrisnan method.
+- `VENKATAKRISHNAN_WANG` : Slope limiter using Venkatakrisnan method, with the small non-vanishing bias to prevent divisions by zero based on the min-to-max range of the solution.
```

Comment:
```suggestion
- `VENKATAKRISHNAN_WANG` : Slope limiter using Venkatakrisnan method, with a small non-vanishing bias proportional to the min-to-max range of the solution, to avoid limiting in small regions.
```